packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1660,20 +1660,18 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const savedFlushMode = this.flushMode;
         this.setFlushMode(FlushMode.TurnBased);
 
-        try {
-            this.trackOrderSequentiallyCalls(callback);
-            this.flush();
-            this.setFlushMode(savedFlushMode);
-        } catch(error) {
-            this.closeFn(CreateProcessingError(error, "orderSequentially"));
-            throw error; // throw the original error for the consumer of the runtime
-        }
+        this.trackOrderSequentiallyCalls(callback);
+        this.flush();
+        this.setFlushMode(savedFlushMode);
     }
 
     private trackOrderSequentiallyCalls(callback: () => void): void {
         try {
             this._orderSequentiallyCalls++;
             callback();
+        } catch (error) {
+            this.closeFn(CreateProcessingError(error, "orderSequentially"));
```

Comment:
this isn't really a processing error. processing errors are really more about the processing of incoming state, like ops or snapshot data. not sure what the "right" error would be here

Reply:
My thinking is that it's an error in 'processing' a batch. Let me know if you have strong opinions on this. I was split between this one and `UsageError` but I didn't like the latter as it's not like the client would intentionally misuse this API by throwing exceptions...